spec/mailers/emailer_spec.rb

```diff
@@ -6,6 +6,19 @@
   include EmailSpec::Matchers
   fixtures :affiliates, :users, :features, :memberships
 
+  describe '#account_deactivation_warning' do
```

Comment:
Class methods should be prepended with a period: `describe '.account_deactivation_warning'` (I see the other class methods in this spec are prepended with a `#`, but that ain't right)
http://www.betterspecs.org/#describe

Reply:
ok I was trying to just be consistent with what is currently there.